app/uk/gov/hmrc/agentsubscriptionfrontend/controllers/CheckAgencyController.scala

```diff
@@ -35,6 +35,15 @@ import uk.gov.hmrc.play.frontend.controller.FrontendController
 
 import scala.concurrent.Future
 
+object CheckAgencyController {
```

Comment:
I think this is established pattern in Scala to keep public final immutable stuff in companion object having same name as a class? KnownFactsForm doesn't depend on anything else in controller so we should keep testing it simple as possible.